app/authorization/server/methods/saveRole.js

```diff
@@ -19,6 +20,13 @@ Meteor.methods({
 				method: 'authorization:saveRole',
 			});
 		}
+		const roles = getRoles();
+
+		if (roles.some((role) => role.name === roleData.name.toLowerCase())) {
```

Comment:
Instead of fetching all the roles, it would be better to query the database with `roleData.name` and see if that returns something.

Reply:
Hello, @KevLehman I have made the changes and added a commit as you requested. Thanks. Please review